src/generic.rs

```diff
@@ -527,5 +527,6 @@ where
     }
 }
 
+// Safety: Same bounds as `std::borrow::Cow`.
 unsafe impl<T: Beef + Sync + ?Sized, U: Capacity> Sync for Cow<'_, T, U> {}
-unsafe impl<T: Beef + Send + ?Sized, U: Capacity> Send for Cow<'_, T, U> {}
+unsafe impl<T: Beef + Sync + ?Sized, U: Capacity> Send for Cow<'_, T, U> {}
```

Comment:
```suggestion
unsafe impl<T: Beef + Send + Sync + ?Sized, U: Capacity> Send for Cow<'_, T, U> {}
```

Reply:
Aah you're right, this is what the std library uses too https://doc.rust-lang.org/beta/alloc/borrow/enum.Cow.html#impl-Send